### ISSUE TITLE
[Bug Report][2.5.8] Button does not lose focus after closing the popup

### ISSUE THREAD

**Issue by Mikilll94** [vuetifyjs/vuetify#14201]
### Environment
**Vuetify Version:** 2.5.8
**Vue Version:** 2.6.14
**Browsers:** Chrome 93.0.4577.82
**OS:** Windows 10

### Steps to reproduce
1. Click the button.
2. Close the popup.
3. Look at the color of the button. It did not lose the focus.

### Expected Behavior
Button should be unfocused after closing the popup.

### Actual Behavior
Button is still focused after closing the popup.

### Reproduction Link
<a href="https://codepen.io/mikilll94/pen/abwQGpN" target="_blank">https://codepen.io/mikilll94/pen/abwQGpN</a>

### Other comments
This is probably a regression. It was working in the previous versions.

**Comment by glen-84** [comment]
Why should it be unfocussed? It was the last element with focus before opening the dialogue?

**Comment by Mikilll94** [comment]
@glen-84 Yes, but I clicked on the background so it should unfocus. IMO it's normal behavior to unfocus elements when you click on the background of the page.

**Comment by glen-84** [comment]
Clicking on the overlay/backdrop is not the same as clicking on the background of the page. Clicking on the background does remove the focus from the button.

After closing the dialogue (regardless of method), the button should remain focussed so that the dialogue can be reopened via the keyboard.
